test/unit/Tss2_Sys_Initialize.c

```diff
@@ -0,0 +1,191 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include <setjmp.h>
+#include <cmocka.h>
+
+#include "sapi/tpm20.h"
+#include "sysapi_util.h"
+
+TSS2_TCTI_CONTEXT *resMgrTctiContext = 0;
+
+/**
+ * Sets tssCreator to weird value
+ */
+static void
+Tss2_Sys_Initialize_tssCreator_unit (void **state)
+{
+    UINT32 contextSize = 1000;
+    TSS2_RC rval;
+    TSS2_SYS_CONTEXT *sysContext;
+    TSS2_ABI_VERSION tstAbiVersion = { 0xF0000000, TSS_SAPI_FIRST_FAMILY, TSS_SAPI_FIRST_LEVEL, TSS_SAPI_FIRST_VERSION };
+
+    contextSize = Tss2_Sys_GetContextSize(contextSize);
+    sysContext = (TSS2_SYS_CONTEXT *)malloc(contextSize);
+    assert_non_null(sysContext);
+    rval = Tss2_Sys_Initialize(sysContext, contextSize, resMgrTctiContext, &tstAbiVersion);
+    assert_int_equal(rval, TSS2_SYS_RC_ABI_MISMATCH);
```

Comment:
I'm not sure if this should be not_equal or if I wrote these tss_ tests wrong. Because this is what I get from the logs.

```
test/unit/Tss2_Sys_Initialize.log
[==========] Running 10 test(s).
[ RUN      ] Tss2_Sys_Initialize_tssCreator_unit
80005 != 80004
../test/unit/Tss2_Sys_Initialize.c:27: error: Failure!
[  FAILED  ] Tss2_Sys_Initialize_tssCreator_unit
[ RUN      ] Tss2_Sys_Initialize_tssFamily_unit
80005 != 80004
../test/unit/Tss2_Sys_Initialize.c:46: error: Failure!
[  FAILED  ] Tss2_Sys_Initialize_tssFamily_unit
[ RUN      ] Tss2_Sys_Initialize_tssLevel_unit
80005 != 80004
../test/unit/Tss2_Sys_Initialize.c:65: error: Failure!
[  FAILED  ] Tss2_Sys_Initialize_tssLevel_unit
[ RUN      ] Tss2_Sys_Initialize_tssVersion_unit
80005 != 80004
../test/unit/Tss2_Sys_Initialize.c:84: error: Failure!
[  FAILED  ] Tss2_Sys_Initialize_tssVersion_unit
[ RUN      ] Tss2_Sys_Initialize_argCombo1_unit
[       OK ] Tss2_Sys_Initialize_argCombo1_unit
[ RUN      ] Tss2_Sys_Initialize_argCombo2_unit
[       OK ] Tss2_Sys_Initialize_argCombo2_unit
[ RUN      ] Tss2_Sys_Initialize_argCombo3_unit
[       OK ] Tss2_Sys_Initialize_argCombo3_unit
[ RUN      ] Tss2_Sys_Initialize_argCombo4_unit
[       OK ] Tss2_Sys_Initialize_argCombo4_unit
[ RUN      ] Tss2_Sys_Initialize_tctiContext1_unit
[       OK ] Tss2_Sys_Initialize_tctiContext1_unit
[ RUN      ] Tss2_Sys_Initialize_tctiContext2_unit
[       OK ] Tss2_Sys_Initialize_tctiContext2_unit
[==========] 10 test(s) run.
[  PASSED  ] 6 test(s).
[  FAILED  ] 4 test(s), listed below:
[  FAILED  ] Tss2_Sys_Initialize_tssCreator_unit
[  FAILED  ] Tss2_Sys_Initialize_tssFamily_unit
[  FAILED  ] Tss2_Sys_Initialize_tssLevel_unit
[  FAILED  ] Tss2_Sys_Initialize_tssVersion_unit
```